src/colorize.rs

```diff
@@ -0,0 +1,84 @@
+use colored::Colorize;
+
+pub enum STL {
+    Red,
+    Green,
+    Blue,
+    Yellow,
+    White,
+    Black,
+    Cyan,
+    Magenta,
+    BrightRed,
+    BrightGreen,
+    BrightBlue,
+    BrightYellow,
+    BrightWhite,
+    BrightBlack,
+    BrightCyan,
+    BrightMagenta,
+    OnRed,
+    OnGreen,
+    OnBlue,
+    OnYellow,
+    OnWhite,
+    OnBlack,
+    OnCyan,
+    OnMagenta,
+    OnBrightRed,
+    OnBrightGreen,
+    OnBrightBlue,
+    OnBrightYellow,
+    OnBrightWhite,
+    OnBrightBlack,
+    OnBrightCyan,
+    OnBrightMagenta,
+    Normal,
+    TrueColor(u8, u8, u8),
+
+    Bold,
+}
+
+pub fn styled_output(s: &str, styles: Vec<STL>) {
```

Comment:
In code bellow, you are printing every part separately, which is not a good practice. It is fine for that task, you should not change it, but keep it in mind.